wix/main.wxs

```diff
@@ -0,0 +1,240 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<!--
+  Copyright (C) 2017 Christopher R. Field.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!--
+  Please do not remove these pre-processor If-Else blocks. These are used with
+  the `cargo wix` subcommand to automatically determine the installation
+  destination for 32-bit versus 64-bit installers. Removal of these lines will
+  cause installation errors.
+-->
+<?if $(var.Platform) = x64 ?>
+    <?define Win64 = "yes" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFiles64Folder" ?>
+<?else ?>
+  <?define Win64 = "no" ?>
+  <?define PlatformProgramFilesFolder = "ProgramFilesFolder" ?>
+<?endif ?>
+
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>
+
+    <Product
+        Id='*'
+        Name='Amazon QLDB Shell'
+        UpgradeCode='84F8D807-9709-40A0-A8BA-ED5CBE94A74F'
+        Manufacturer='Marc Bowes'
```

Comment:
Please change this (I don't know what to, but not me.) Please update Cargo.toml at the same time.

Reply:
Changed to Amazon.  Cargo wix init initially generated the wxs file.

Reply:
We need to update the [cargo.toml ](https://github.com/awslabs/amazon-qldb-shell/blob/rust/Cargo.toml#L4) file. Wix is getting the author from that file.

Also I think it should be Amazon Web Services to match the [Python shell](https://github.com/awslabs/amazon-qldb-shell/blob/master/setup.py#L30)

Reply:
We don't need the change the cargo.toml since we don't generate the main.wxs file from there anymore.

Reply:
Still need to update the cargo.toml file since its used for packaging

Reply:
Changed the names to Amazon Web Services in both cargo.toml and main.wxs files.